src/constructor.io/models/Catalog/RetrieveSearchabilitiesRequest.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Net.Http;
+using Constructorio_NET.Utils;
+
+namespace Constructorio_NET.Models
+{
+    /// <summary>
+    /// Constructor.io Retrieve Searchabilities Request Class.
+    /// </summary>
+    public class RetrieveSearchabilitiesRequest
+    {
+        /// <summary>
+        /// Gets or sets name of searchability field to retrieve.
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets the page of searchability results to retreive. Can't be used with Offset.
+        /// </summary>
+        public int Page { get; set; }
+
+        /// <summary>
+        /// Gets or sets the offset of results to skip. Can't be used with Page.
+        /// </summary>
+        public int Offset { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the section.
+        /// </summary>
+        public string Section { get; set; }
+
+        /// <summary>
+        /// Gets or sets the number of results per page to retrieve.
+        /// </summary>
+        public int NumResultsPerPage { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to retrieve only results which are either exact_searchable or fuzzy_searchable.
+        /// </summary>
+        public bool Searchable { get; set; }
+
+        /// <summary>
+        /// Gets or sets a the criteria by which searchability results should be sorted. Valid criteria is "name".
+        /// </summary>
+        public string SortBy { get; set; }
+
+        /// <summary>
+        /// Gets or sets a the order by which searchability results should be sorted. Valid criteria is "descending" or "ascending".
+        /// </summary>
+        public string SortOrder { get; set; }
+
+        /// <summary>
+        /// Gets or sets filters used to refine results. Valid keys are "name", "exact_searchable", or "fuzzy_searchable".
+        /// </summary>
+        public Dictionary<string, string> Filters { get; set; }
+
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RetrieveSearchabilitiesRequest"/> class.
+        /// </summary>
+        public RetrieveSearchabilitiesRequest()
+        {
+        }
+
+        /// <summary>
+        /// Get request parameters.
+        /// </summary>
+        /// <returns>Hashtable of request parameters.</returns>
+        public Hashtable GetRequestParameters()
+        {
+            Hashtable parameters = new Hashtable();
+
+            if (!string.IsNullOrEmpty(this.Name))
```

Comment:
When you add it to the name field it gives the response in the different format. So I decided to put it in the filters[name] field since it would give the same format, be functionally the same.